tests/libnet/cloudinit.go

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package libnet
+
+import (
+	"sigs.k8s.io/yaml"
+)
+
+type CloudInitNetworkData struct {
+	Version   int                           `yaml:"version"`
+	Ethernets map[string]CloudInitInterface `yaml:"ethernets,omitempty"`
+}
+
+type CloudInitInterface struct {
+	AcceptRA       *bool                `yaml:"accept-ra,omitempty"`
```

Comment:
do you want to add matchers support ? or not in this PR ?

Reply:
As we do not use this at the moment, I prefer to do it only when needed.
Unlike the other parameters, a matcher is a special behavior which controls on which interface the configuration is bind to.